works/views.py

```diff
@@ -261,7 +261,7 @@ def publication_edit(request, publication_id=None):
             series = SeriesToWorkFomSet(request.POST, request.FILES, instance=instance)
```

Comment:
I'd rename the variables 'series', 'creators' etc. to `series_to_works`, `creator_to_works`, etc., to make it clearer that we're not doing `series.delete` here.